components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -0,0 +1,2222 @@
+            }catch (SecurityViolation sv) {
+            }
+        /* take care of workflows which do not use sudo */
+        if (!isSudoing) {
+            loginUser(lightAdmin);
+        }
+        /*in order to find the image in whatever group, get context with group
+         * set to -1 (=all groups) */
+        client.getImplicitContext().put("omero.group", Long.toString(-1));
+        /* try to move the image into another group of the normalUser
+         * which should succeed if sudoing and also in case
+         * the light admin has Chgrp permissions
+         * (i.e. isExpectSuccessInMemberGroup is true) */
+        doChange(client, factory, Requests.chgrp().target(image).toGroup(normalUsersOtherGroupId).build(), isExpectSuccessInMemberGroup);
+        /* note in which group the image and the original file are now */
+        long afterFirstChgrpImageGroupId = ((RLong) iQuery.projection(
+                "SELECT details.group.id FROM Image i WHERE i.id = :id",
+                new ParametersI().addId(image.getId())).get(0).get(0)).getValue();
+        remoteFileGroupId = ((RLong) iQuery.projection(
+                "SELECT details.group.id FROM OriginalFile o WHERE o.id = :id",
+                new ParametersI().addId(remoteFileId)).get(0).get(0)).getValue();
+        if (isExpectSuccessInMemberGroup) {
+            Assert.assertEquals(afterFirstChgrpImageGroupId, normalUsersOtherGroupId);
+            Assert.assertEquals(remoteFileGroupId, normalUsersOtherGroupId);
+        } else {
+            Assert.assertEquals(afterFirstChgrpImageGroupId, normalUser.groupId);
+            Assert.assertEquals(remoteFileGroupId, normalUser.groupId);
+        }
+        /* in any case, the image should still belong to normalUser */
+        assertOwnedBy(image, normalUser);
+
+        /* try to move into another group the normalUser
+         * is not a member of, which should fail in all cases
+         * except the light admin has Chgrp permission and is not sudoing
+         * (i.e. chgrpNoSudoExpectSuccessAnyGroup is true) */
+        doChange(client, factory, Requests.chgrp().target(image).toGroup(anotherGroupId).build(),
+                chgrpNoSudoExpectSuccessAnyGroup);
+        long afterSecondChgrpImageGroupId = ((RLong) iQuery.projection(
+                "SELECT details.group.id FROM Image i WHERE i.id = :id",
+                new ParametersI().addId(image.getId())).get(0).get(0)).getValue();
+        remoteFileGroupId = ((RLong) iQuery.projection(
+                "SELECT details.group.id FROM OriginalFile o WHERE o.id = :id",
+                new ParametersI().addId(remoteFileId)).get(0).get(0)).getValue();
+        if(chgrpNoSudoExpectSuccessAnyGroup) {
+            /* check that the image moved to another group */
+            Assert.assertEquals(afterSecondChgrpImageGroupId, anotherGroupId);
+            Assert.assertEquals(afterSecondChgrpImageGroupId, anotherGroupId);
```

Comment:
This is fixed in https://github.com/openmicroscopy/openmicroscopy/pull/5275/files#diff-d91232ef1e245a87e311c4a79a2a5f77R792 and https://github.com/openmicroscopy/openmicroscopy/pull/5275/files#diff-d91232ef1e245a87e311c4a79a2a5f77R914 - the testChgp was split in the latter PR into two separate tests, the check for the annotations was added as well as a check for the OriginalFile.